lib/Server.js

```diff
@@ -345,7 +345,11 @@ class Server {
 
     if (Array.isArray(contentBase)) {
       contentBase.forEach((item) => {
-        this.app.get('*', express.static(item));
+        if (this.options.contentBasePrependPublic) {
+          this.app.use(this.options.publicPath, express.static(item));
+        } else {
+          this.app.get('*', express.static(item));
+        }
```

Comment:
And still breaking change

Reply:
you mean a new `string` option should added special for `contentBase`?

Reply:
and just have `contentBasePublicPath` defaults to `*`
```
this.app.get(contentBasePublicPath, express.static(item));
```

Reply:
Can't understand you, as you written above you need setup `publicPath` for `contentBase` (let's name this `contentBasePublicPath`) based on you logic
```js
if (this.options.contentBasePrependPublic) {
  this.app.use(this.options.publicPath, express.static(item));
}
```

You want use publicPath for static resources, it very misleading, we should not mix logic `publicPath` and potential new option public path for `contentBase`, we should use difference option for this.

Also i want to warn you what it is very strange structure and can be dangerous in some cases. Imagine you have app with user profile and allow peoples update/upload their avatars, you should have 2 directories - `dist` for bundle and `static` for uploaded images. Mix them can be complicate deploy your app and may cause security problems (when you have invalid configuration for webserver, believe me it's a lot :smile: ). Anyway we open for this open

Reply:
>  as you written above you need setup publicPath for contentBase (let's name this contentBasePublicPath)

yes

> we should not mix logic publicPath and potential new option public path for contentBase, we should use difference option for this.

agree

let me just make some tests.

Reply:
:+1: